Cargo.toml

```diff
@@ -9,6 +9,9 @@ resolver = "2"
 repository = "https://github.com/soenkehahn/agora"
 homepage = "https://github.com/soenkehahn/agora"
 
+[workspace]
+members = [".", "bin/prerelease"]
```

Comment:
This is kind of nice. I think having a new package for a small script seems a bit overkill, but I don't know a better way. And this is less intrusive than I thought it'd be.

I mentioned `rust-script` before, which makes scripts more self-contained, but then I wouldn't know how to write tests for those scripts. So this LGTM.